### PR TITLE
[IOTDB-3469] Support set TTL in new cluster

### PR DESCRIPTION
**Pull request by 23931017wu**

### CODE REVIEW

**Review by SpriCoder — APPROVED** [review]
LGTM~

**Review by mychaow — APPROVED** [review]
LGTM

**Comment by HTHou** [comment]
Hi, does `set TTL` work in new standalone IoTDB as well?

**Comment by 23931017wu** [comment]
> Hi, does `set TTL` work in new standalone IoTDB as well?

Sorry, not yet.

**Comment by 23931017wu** [comment]
> Hi, does `set TTL` work in new standalone IoTDB as well?

I'll finish it later

File: confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java [hunk 1]
```diff
@@ -727,4 +727,8 @@ public void addMetrics() {
     partitionManager.addMetrics();
     nodeManager.addMetrics();
   }
+
+  public boolean ifStorageGroupExist(String storageGroup) {
+    return clusterSchemaManager.getStorageGroupNames().contains(storageGroup.toString());
```
Comment (qiaojialin) [inline confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java:732]: add confirmLeader()

File: confignode/src/main/java/org/apache/iotdb/confignode/manager/PartitionManager.java [hunk 2]
```diff
@@ -424,6 +425,9 @@ private void waitRegionCreation(List<String> waitingList, TConsensusGroupType co
     throw new TimeoutException("");
   }
 
+  public Set<TDataNodeLocation> getDataNodeLocation(String storageGroup, TConsensusGroupType type) {
```
Comment (qiaojialin) [inline confignode/src/main/java/org/apache/iotdb/confignode/manager/PartitionManager.java:428]: add related function in ConfigManager with confirmLeader()

File: confignode/src/main/java/org/apache/iotdb/confignode/service/thrift/ConfigNodeRPCServiceProcessor.java [hunk 3]
```diff
@@ -218,7 +221,32 @@ public TSStatus deleteStorageGroups(TDeleteStorageGroupsReq tDeleteReq) throws T
...
+    if (!configManager.ifStorageGroupExist(req.storageGroup)) {
+      return RpcUtils.getStatus(
+          TSStatusCode.STORAGE_GROUP_NOT_EXIST,
+          "storageGroup " + req.storageGroup + " does not exist");
+    }
+    TSStatus tsStatus = new TSStatus();
+    tsStatus = configManager.setTTL(new SetTTLReq(req.getStorageGroup(), req.getTTL()));
+    Set<TDataNodeLocation> dataNodeLocations =
+        configManager
+            .getPartitionManager()
```
Comment (qiaojialin) [inline confignode/src/main/java/org/apache/iotdb/confignode/service/thrift/ConfigNodeRPCServiceProcessor.java:233]: call the getDataNodeLocatoinList in ConfigManager with confirmLeader()

We need to read/write partition information through ConfigNode manager currently.

**Review by qiaojialin — COMMENTED** [review]
Looks well in total, only a little I'm not sure about the confirmLeader()

**Review by neuyilan — APPROVED** [review]
LGTM

**Review by HTHou — APPROVED** [review]

**Review by qiaojialin — APPROVED** [review]

**Pull request merged**